cmd/utils/flags.go

```diff
@@ -507,6 +508,21 @@ var (
 		Usage: "HTTP path path prefix on which JSON-RPC is served. Use '/' to serve on all paths.",
 		Value: "",
 	}
+	HTTPRequestReadTimeout = cli.IntFlag{
+		Name:  "http.timeout.read",
+		Usage: "Timeout in seconds for HTTP-RPC read requests",
+		Value: int(rpc.DefaultHTTPTimeouts.ReadTimeout),
```

Comment:
If the message timeout is in seconds, this `value` should be `int(rpc.DefaultHTTPTimeouts.ReadTimeout / Time.Second)`